### PR TITLE
Jumbotron: Putting heading font size in variables to allow for easier overrides

### PR DESCRIPTION
**Pull request by max-b**
This was relevant on one of my projects recently. It'd be nice to be able to change the jumbotron heading font-size with a simple variable override.

Thanks!

### CODE REVIEW

File: less/jumbotron.less [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
     h1,
     .h1 {
-      font-size: (@font-size-base * 4.5);
+      font-size: @jumbotron-font-size;
```
Comment (cvrebert) [inline less/jumbotron.less]: Presumably you meant for this to be `@jumbotron-heading-font-size` ?

File: less/variables.less [hunk 2]
```diff
@@ -483,7 +483,8 @@
 @jumbotron-color:                inherit;
 @jumbotron-bg:                   @gray-lighter;
 @jumbotron-heading-color:        inherit;
-@jumbotron-font-size:            ceil((@font-size-base * 1.5));
+@jumbotron-heading-font-size:    ceil((@font-size-base * 1.5));
```
Comment (cvrebert) [inline less/variables.less]: Presumably the values for these 2 vars were accidentally swapped?

Comment (max-b) [inline less/variables.less]: phooey - yeah you're correct. Should I just scrap this commit/request and submit a new one, or make another commit and add it to the request?

Comment (cvrebert) [inline less/variables.less]: Might as well just make a whole new one.

**Comment by max-b** [comment]
I fixed the stupid mistake that @cvrebert pointed out - thanks for your patience!

**Comment by cvrebert** [comment]
CC: @mdo

**Comment by mdo** [comment]
:shipit:
